README.md

```diff
@@ -13,12 +13,34 @@ YES | mastergberry | jovo123, jcryan | A fish animation
 ```
 ================
 
-F16 final remarks
+# F16 final remarks
 
-Currently the game runs, and we have the following classes:
+### Currently the game runs, and we have the following classes:
 * Game logics: Score, Score Manager, ScoreComparer, ScumOftheSea, JellyFish, Fish, Shark, SoundEffects, Character.
 * GUI: Menu, GeneralPathWrapper.
 * Mixed: FishAnimationEnvironment.
 
 
 The main function to start the program is in Menu.java. Menu.java takes care of building the main menu of the game, and some Jbuttons on it are tied to some functions in FishAnimationEnvironment.java, where the bulk of the game is defined. Currently, FishAnimationEnvironemnt.java is a mix of Game Logics and GUI stuffs, so it is pretty hard to read through and understand. A code refactoring on this java file is highly recommanded: try to separate the game logic from GUI. The Score Class implements Serializable so that the scores can be stored and provides that game-stop feature, but it is more usual to record the highscores in a database. It is challanging, but a nice chance to learn how to drive the database inside a Java program using JDBC. 
+
+# F17 update
+
+### Welcome to Fish Animation
+* Hello! The goal of the game is to eat as many fish as you can
+* To help developers add more features, the game is set to win whenver you get 50 points
+* There's no time limit, but watch out! Jellyfish are out to get you!
+* If you eat the jellyfish, you'll lose health!
+* If your health is 0 or you get too many negative points, you lose.
+* There are three different difficulties, so test your skills on all of them! Good eating!
+
+
+### How to Play
+* To eat the fish
+* click on the shark and hold down the mouse button to move your shark's mouth to the fish.
+* You can also use the arrow keys to move the fish as well."
```

Comment:
- This should be "shark," not "fish."
 - There's a stray `"` at the end of this one.
 - The next four lines are kind of obvious.  Let's remove them.

Reply:
fixed